src/PHPUnit/Controller/AbstractControllerTestCase.php

```diff
@@ -304,6 +304,14 @@ public function dispatch($url, $method = null, $params = [], $isXmlHttpRequest =
 
         $this->url($url, $method, $params);
         $this->getApplication()->run();
+        
+        if (true !== $this->traceError) {
+            return;
+        }
+        $exception = $this->getApplication()->getMvcEvent()->getParam('exception');
+        if ($exception instanceof \Exception) {
```

Comment:
I would suggest to change here to have:
```php
if ($exception instanceof \Throwable || $exception instanceof \Exception)
```

so we can support all Throwables on PHP 7.0+.